homework/grade2/Tom-grade-2.py

```diff
@@ -0,0 +1,41 @@
+# homework for grade 2
+def stringornumber():
+
+    check = input('Enter a number or string: ')
+
+    if check.isalpha():
+        print('{} is a string'.format(check))
+    elif check.isnumeric():
+        print('{} is a number'.format(str(check)))
+    else:
+        print('{} characters and numbers'.format(check))
+
+def doathing(number):
+
+    control = number
+
+    if number.isnumeric():
+        if len(number) < 50:
+            print('Lets count down')
+            while int(control) > 0:
+                print(control)
+                control = int(control) - 1
+    else:
+        print('You must enter a number')
+        doathing(input('Please enter a number less then 100: '))
+
+def lastthing(somestring):
```

Comment:
For good practice can you add pydocs (Grade 3) to this? It will make it easier to understand.

Reply:
https://realpython.com/documenting-python-code/